src/shared/scrapers/USA/PA/index.js

```diff
@@ -179,6 +180,22 @@ const scraper = {
       this.type = 'table';
       const $ = await fetch.page(this.url);
       const $countyTable = $('td:contains("County")').closest('table');
+
+      const rules = {
+        headings: {
+          0: /county/i,
+          1: /number of cases/i,
+          2: /deaths/i
+        },
+        data: [
+          { column: 0, row: 'ANY', rule: /Adams/ },
+          { column: 1, row: 'ALL', rule: /^[0-9]+$/ },
+          { column: 2, row: 'ALL', rule: /(^[0-9]+|)$/ }
+        ]
+      };
+      const opts = { includeErrCount: 5, logToConsole: true };
+      HtmlTableValidator.throwIfErrors(rules, $countyTable, opts);
```

Comment:
Based on the usage I'm seeing, we should rewrite the class as a function. This will simplify the code, and allow you to specify which functions can be imported in other parts of the codebase

Reply:
That change would be quick.  If that’s the team preference can do.  That public static was just a helper func to simplify the API and usage, but the class itself has hardly any state IIRC.

Reply:
I'm going to assume that this is what the team wants and will make this change, thanks @qgolsteyn.

Reply:
Let me know once you are done this change and I'm happy to review again

Reply:
Pushed @qgolsteyn, thank you!  I only exported a few methods to keep the module API as clean as possible, others can export things if they need.

Reply:
@jzohrab are we still migrating from a class to a function?

Reply:
@camjc that snippet is outdated, everything is a function now.  The code now reads

```
import * as htmlTableValidation from '../../../lib/html/table-validation.js';
...
htmlTableValidator.throwIfErrors(rules, $countyTable, opts);
```